### ISSUE TITLE
deadpool-redis-cluster missing in crates.io

### ISSUE THREAD

**Issue by 2ico** [bikeshedder/deadpool#240]
The README on the master branch links to [https://crates.io/crates/deadpool-redis-cluster](https://crates.io/crates/deadpool-redis-cluster), which is broken at the moment.

**Comment by graysonhead** [comment]
I also noticed this, would be nice to have it fixed.

As an FYI to anyone who doesn't already know, you can stick this in your Cargo.toml for a workaround:
```
deadpool-redis-cluster = { git = "ssh://git@github.com/bikeshedder/deadpool.git", features = ["rt_tokio_1"] }
```

**Comment by bikeshedder** [comment]
I'm very sorry for the delay. I'll try to release `deadpool 0.10` over the weekend and version `0.1` of this crate as well.

**Comment by ginkcode** [comment]
Any good news yet? 😊

**Comment by shoichi1023** [comment]
When will this Crate be released?

**Comment by bikeshedder** [comment]
Sorry for all the delay. I just released `deadpool 0.10` and all the `deadpool-*` crates.

`deadpool-redis-cluster` is now uploaded to crates.io:
- https://crates.io/crates/deadpool-redis-cluster/0.1.0

**Comment by ginkcode** [comment]
Thanks. Nice work!
